Assets/Scripts/DataController.cs

```diff
@@ -144,20 +143,17 @@ void Start()
                     Vector3 magScale = starObject.transform.localScale * magScaleValue;
                     starObject.transform.localScale = magScale;
                     starObject.transform.LookAt(this.transform);
-
+                    // Eventually store constellation color and observed star color separately
+                    newStar.SetStarColor(constellationColor, Color.white);
                     // color by constellation
                     if (colorByConstellation == true)
                     {
-                        changeStarColor(starObject, constellationColor);
-                        newStar.starColor = constellationColor;
+                        Utils.SetObjectColor(starObject, constellationColor);
                     }
```

Comment:
There's probably some other stuff we can add to a Utils module.  Thanks for adding this!